### PR TITLE
Feature/task popup

### PR DESCRIPTION
**Pull request by jhn322**
- Improved styling 
- @media for Mobile

### CODE REVIEW

File: src/components/TaskPopup.jsx [hunk 1]
```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
...
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
+              <button className={`${styles.ExtraBtn} ${styles.DeadlineBtn}`}>
+                Deadline
+              </button>
+              <div className={styles.FlexBtn}>
```
Comment (Regni) [inline src/components/TaskPopup.jsx:75]: This button is wrapped in a div, and makes the styling on mobile look weird.

image

File: src/components/TaskPopup.jsx [hunk 2]
```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
...
+                <button className={`${styles.ExtraBtn} ${styles.DoDateBtn}`}>
+                  Do Date
+                </button>
+              </div>
+              <button className={`${styles.ExtraBtn} ${styles.MemberBtn}`}>
+                Member
+              </button>
+              <div className={styles.FlexBtn}>
```
Comment (Regni) [inline src/components/TaskPopup.jsx:83]: same as the 1 before

**Review by Regni — COMMENTED** [review]
added some comments if this is intended then it is good!

File: src/components/TaskPopup.jsx [hunk 3]
```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
...
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
+              <button className={`${styles.ExtraBtn} ${styles.DeadlineBtn}`}>
+                Deadline
+              </button>
```
Comment (Regni) [inline src/components/TaskPopup.jsx:74]: This button is wrapped in a div, and makes the styling on mobile look weird.

image

File: src/components/TaskPopup.jsx [hunk 4]
```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
...
+              <div className={styles.FlexBtn}>
+                <button className={`${styles.ExtraBtn} ${styles.DoDateBtn}`}>
+                  Do Date
+                </button>
+              </div>
+              <button className={`${styles.ExtraBtn} ${styles.MemberBtn}`}>
+                Member
+              </button>
```
Comment (Regni) [inline src/components/TaskPopup.jsx:82]: same as 1 before

File: src/components/TaskPopup.jsx [hunk 5]
```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
...
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
+              <button className={`${styles.ExtraBtn} ${styles.DeadlineBtn}`}>
+                Deadline
+              </button>
+              <div className={styles.FlexBtn}>
+                <button className={`${styles.ExtraBtn} ${styles.DoDateBtn}`}>
+                  Do Date
+                </button>
+              </div>
```
Comment (Regni) [inline src/components/TaskPopup.jsx]: this button is wrapped in a div and makes style look weird:

image

File: src/components/TaskPopup.jsx [hunk 6]
```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
...
+              <button className={`${styles.ExtraBtn} ${styles.MemberBtn}`}>
+                Member
+              </button>
+              <div className={styles.FlexBtn}>
+                <button className={`${styles.ExtraBtn} ${styles.Extra1Btn}`}>
+                  Extra 1
+                </button>
+              </div>
```
Comment (Regni) [inline src/components/TaskPopup.jsx]: same as before

Comment (jhn322) [reply src/components/TaskPopup.jsx]: Experimenterade lite, gjorde layouten mer simpel nu. Ser bra ut på iPhone (kan variera på andra mobiler).

**Review by EllenSandqvist — APPROVED** [review]
Ser bra ut både på desktop och mobil

**Pull request merged**